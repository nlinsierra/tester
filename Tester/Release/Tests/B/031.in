2H 2C 5S QS 4S 9H KD AD 5H 6D 7C 8C 8S 7S JD TD 4H TC 5D 8H 6H QH
