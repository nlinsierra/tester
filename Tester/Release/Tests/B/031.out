30
3C 4C 5C 6C 9C JC QC KC AC 2D 3D 4D 7D 8D 9D QD 3H 7H TH JH KH AH 2S 3S 6S 9S TS JS KS AS
